APCAppCore/APCAppCore/UI/TabBarControllers/Profile/APCChangePasscodeViewController.h

```diff
@@ -32,12 +32,15 @@
 // 
  
 #import <UIKit/UIKit.h>
-#import "APCPasscodeView.h"
+
+@class APCPasscodeView;
+
 
 @interface APCChangePasscodeViewController : UIViewController
 
-@property (weak, nonatomic) IBOutlet UILabel *textLabel;
-@property (weak, nonatomic) IBOutlet APCPasscodeView *passcodeView;
+@property (weak, nonatomic, nullable) IBOutlet UILabel *textLabel;
```

Comment:
I left it out because there is nothing else in those headers. But yes, makes sense to add them.